docs/Drawer.md

```diff
@@ -9,6 +9,7 @@ The `<pxb-drawer>` is used to organize content (typically navigation links) with
 ### Drawer Usage
 
 ```html
+// your-component.html
```

Comment:
What's the deal with these comments...I thought we had a PR that converted all of these to HTML syntax?